forge/__init__.py

```diff
@@ -29,20 +35,38 @@ def register_plugin(self, name, plugin, helptext):
         self.registry[name] = (plugin, helptext)
 
     def print_help(self):
+        """ Print Help For All Registered Plugins """
         help_entries = []
         for name in self.registry:
             help_entries.append([name, self.registry[name][1]])
         print(tabulate(help_entries, ['function', 'blurb']))
 
     def execute(self, command, args):
+        """ Execute Plugin """
         if command == 'help':
             self.print_help()
         else:
             self.registry[command][0](args)
 
+    def parse_conf(self, forge_dir, conf_file):
+        """ Parse Plugin Configuration File """
+        config = configparser.ConfigParser()
+        config.read(forge_dir + conf_file)
+
+        if not os.path.exists(forge_dir + conf_file):
+            self._init_conf_file(forge_dir + conf_file, config)
+
+        for key in config['plugin-definitions']:
+            self.plugins.append(forge_dir + 'plugins/' + key)
+
+    @staticmethod
+    def _init_conf_file(path, config):
+        config['plugin-definitions'] = {}
```

Comment:
And you created a resource leak.  Use  **with** to close everything when finished.

Reply:
Will do.